### PR TITLE
Fixed formatting, cleared up writing

### PR DESCRIPTION
**Pull request by DavidLu1997**
Many paragraphs were formatted as multiple single lines.

@crspeller please review :)

### CODE REVIEW

**Comment by it33** [comment]
Hold on, this removes headings from error messages, which

1. Removes anchors to those error messages, which are used to deep-link to the messages on forums
2. Breaks existing links in forums

Propose reverting that part back.

**Comment by crspeller** [comment]
I can't really tell how the formatting looks until it's on docs.mattermost.com. 

@it33 He has changed more than the formatting. Please review the text more closely and merge when ready.

**Comment by DavidLu1997** [comment]
I changed a few lines regarding database manipulation and basic troubleshooting steps in the beginning. Specifically, I made the lines more concise. I can revert if needed.

File: source/install/troubleshooting.rst [hunk 1]
```diff
@@ -6,44 +6,34 @@ Troubleshooting
...
-   - Use the step-by-step install guides before trying variations.
+- Start simple with the step-by-step install guides for your operating system
 
-2. Check the logs for errors
-   - In the directory where you installed Mattermost look in the ``mattermost.log`` file for error messages. Also check NGNIX error logs.
+- Check the logs (``mattermost.log`` and NGNIX logs) for errors
 
-3. Web search the error message
-   - If you have an error message you don't understand, search the web for the message to try and find an existing article.
+- Search error messages, existing solutions can often work
```
Comment (it33) [inline source/install/troubleshooting.rst]: Please specify where to search, many Mattermost users don't have English as a first language, so we need to be extra specific

File: source/install/troubleshooting.rst [hunk 2]
```diff
@@ -6,44 +6,34 @@ Troubleshooting
 Important notes
 ---------------
 
-**DO NOT manipulate the Mattermost database**
+Database
```
Comment (it33) [inline source/install/troubleshooting.rst]: Propose original title, we don't want people to miss this.

**Comment by it33** [comment]
Also, please end bullet points with periods to be consistent with the original doc. Otherwise looks good, nice work!

**Pull request merged**